src/axom/primal/tests/primal_tetrahedron.cpp

```diff
@@ -623,6 +624,39 @@ TEST_F(TetrahedronTest, regularTetrahedron)
     EXPECT_EQ(primal::ON_POSITIVE_SIDE, primal::orientation(pt, tri));
   }
 }
+
+TEST_F(TetrahedronTest, checkSign)
+{
+  using QPoint = TetrahedronTest::QPoint;
+  using QTet = TetrahedronTest::QTet;
+
+  int indices[] = {0, 1, 2, 3};
+
+  for(int i = 0; i < this->numTetrahedra(); ++i)
+  {
+    QTet tet = this->getTet(i);
+    double expVolume = tet.signedVolume();
+
+    // Run sign check through all vertex permutations for the tetrahedron
+    do
+    {
+      QTet tetPermuted =
+        QTet(tet[indices[0]], tet[indices[1]], tet[indices[2]], tet[indices[3]]);
+
+      double preCheckVolume = tetPermuted.volume();
```

Comment:
Shouldn't this call signedVolume instead of volume?

Reply:
> Shouldn't this call signedVolume instead of volume?

This is part of a check to verify the absolute value of the volume is still the same _and_ now positive after calling `checkAndFixOrientation()`, so we want the unsigned volume before calling the function.